examples/hitl/rearrange_v2/collaboration_episode_loader.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Meta Platforms, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import typing
+from typing import Dict
+
+from habitat.datasets.rearrange.rearrange_dataset import RearrangeEpisode
+
+try:
+    from habitat_llm.agent.env import dataset  # noqa: F401
+    from habitat_llm.agent.env.dataset import CollaborationEpisode
+
+    collaboration_episode_enabled = True
+except ImportError:
+    print("Unable to load CollaborationDataset episode format.")
+    collaboration_episode_enabled = False
+
+
+class CollaborationEpisodeData:
+    instruction: str = ""
+    object_goal_pairs: Dict[str, str] = {}
```

Comment:
```suggestion
    #dict mapping a pickable object instance handle to the object instance handle for the furniture it is placed upon
    object_goal_pairs: Dict[str, str] = {}
```
Why is this called object "goal" pairs? Looks like it is populated from the initial state of the objects, not the goal state. :thinking: